### PR TITLE
Added example to afterSaveCommit

### PR DESCRIPTION
**Pull request by yosus**
Example to clearly show how to get $options from save

### CODE REVIEW

**Comment by ndm2** [comment]
Is there a reason that the example would go specifically there, and _only_ there, given that there's many other callbacks with options too?

IMHO it would be more suited in the [Saving Data](https://book.cakephp.org/4/en/orm/saving-data.html#saving-entities) page, and/or in a dedicated subsection under [Lifecycle Callbacks](https://book.cakephp.org/4/en/orm/table-objects.html#lifecycle-callbacks).

**Review by Arhell — COMMENTED** [review]
cc @markstory

File: en/orm/table-objects.rst [hunk 1]
```diff
@@ -304,6 +304,18 @@ saves where database operations are implicitly committed. The event is triggered
 only for the primary table on which ``save()`` is directly called. The event is
 not triggered if a transaction is started before calling save.
```
Comment (markstory) [inline en/orm/table-objects.rst]:
```suggestion
not triggered if a transaction is started before calling save::
```

This line needs to end in `::` or the code block won't be formatted.

**Review by markstory — COMMENTED** [review]
I agree with @ndm2. Accessing options is common to all lifecycle callbacks. I will can relocate the section post merge though if you address the formatting problem.

**Review by yosus — COMMENTED** [review]
moved example to `Lifecycle Callbacks`

**Pull request merged**